src/neo/Plugins/RequiredServicesAttribute.cs

```diff
@@ -1,15 +0,0 @@
-using System;
-
-namespace Neo.Plugins
-{
-    [AttributeUsage(AttributeTargets.Class)]
-    public class RequiredServicesAttribute : Attribute
```

Comment:
How it will be checked the required services, in `OnSystemLoaded`?

Reply:
Now the services have been moved to `NeoSystem`. And the systems share the same plugin instances. So you have to check the services in `OnSystemLoaded` or just use `?.` syntax:

```c#
system.GetService<IWalletProvider>()?.GetWallet();
```